src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/EditDraftApprenticeship.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@model SFA.DAS.EmployerCommitmentsV2.Web.Models.EditDraftApprenticeshipViewModel
+
+@{
+    ViewData["Title"] = "Apprenticeships";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <h1 class="govuk-heading-xl">Edit apprentice details</h1>
+        <div class="govuk-inset-text">
```

Comment:
This inset text is incorrect - but there is no screenshot for us to build against in the ticket